compose/ui/ui/src/skikoTest/kotlin/androidx/compose/ui/platform/SkiaLayerTest.kt

```diff
@@ -254,29 +259,31 @@ class SkiaLayerTest {
             translationX = 60f,
             translationY = 7f,
             rotationX = 45f,
```

Comment:
Can we make tests that makes "trapeze" figure in the result screenshot? I.e. when we rotate 45 degrees, the top should be narrower